test/integration/replication.bats

```diff
@@ -2,8 +2,8 @@
 
 load helpers
 
-# Address on which the store will listen (random port between 8000 and 9000).
-STORE_HOST=127.0.0.1:$(( ( RANDOM % 1000 )  + 8000 ))
+# Address on which the store will listen
+STORE_HOST=127.0.0.1:8500
```

Comment:
Should we make the port of the store host configurable? (But maybe later is OK).